packages/core-loggers/package.json

```diff
@@ -29,7 +29,7 @@
   },
   "repository": "https://github.com/pnpm/pnpm/blob/master/packages/core-loggers",
   "scripts": {
-    "start": "pnpm run tsc -- --watch",
+    "start": "pnpm tsc --watch",
```

Comment:
it should be just `tsc --watch`

there's no need to start an additional pnpm process.

Reply:
replace all cmd here?

Reply:
yes

Reply:
done